docs/authorization/globus_app/apps.rst

```diff
@@ -0,0 +1,45 @@
+.. _globus_apps:
+
+.. currentmodule:: globus_sdk
+
+GlobusApps
+==========
+
+:class:`GlobusApp` is a high level construct designed to simplify the process of
+of authenticating for Globus service clients.
+
+A ``GlobusApp`` requires a programmatic Globus Auth entity called a "client" to be
+supplied as the requester driving authentication flows. A client can be created and
+managed through the `Globus Developer Console <https://developers.globus.org/>`_.
```

Comment:
I'm open to debating this, but I think we should send users to

```suggestion
managed through the `Globus Developer Console <https://app.globus.org/settings/developers>`_.
```

now that it's integrated into the webapp.

Reply:
Yeah, I didn't know which was more desirable; I copied from other locations in the SDK (and have used this link in past PRs) but for the files I've touched as a part of this change I can update to using the app-based one.

Reply:
I just opened a thread in #auth to confirm what they'd like us to do.

Reply:
Confirmed with Brigitte that `https://app.globus.org/settings/developers` should replace all instances of `https://developers.globus.org/`.

Updated all of the usages in this PR.